compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -1051,30 +1050,25 @@ public void visit(BLangFieldBasedAccess fieldAccessExpr) {
         }
 
         BLangVariableReference targetVarRef = fieldAccessExpr;
-        if (fieldAccessExpr.expr.type.tag == TypeTags.ENUM) {
```

Comment:
Shall we remove enum from type tags as well? (TypeTags.ENUM)

Reply:
Removed with 4951ad6